test/trackler/problem_test.rb

```diff
@@ -82,6 +82,10 @@ def test_track_with_no_canonical_data
     assert_equal nil, problem.canonical_data_url
   end
 
+  def test_problem_with_no_metadata_yml
+    refute Trackler::Problem.new('no-metadata', FIXTURE_PATH).exists?
+  end
```

Comment:
> I think you force pushed and orphaned these comments: 7bfbffe

Yeah. This is a mess. I should have been more careful.

> Having x-api rely on fixture data in the trackler gem is really messed* up.

No. No, it's not fucked up it's a trade-off. That I made consciously. For perfectly good reasons.

My options were:

1. Not actually test things.
2. Test using live data which changes every day
3. Test using fixture data which doesn't change a lot